<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PIP_Equipo4_Unidad2_Clase/imagenes/uat.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>690</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../PIP_Equipo4_Unidad2_Clase/imagenes/Equipo4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pregunta">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>721</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 15px solid black;
border-radius: 20px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Area de preguntas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>801</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid white;
color: rgb(255, 255, 255);
border-radius: 15 px;</string>
    </property>
    <property name="text">
     <string>Examen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>791</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid white;
color: rgb(255, 255, 255);
border-radius:15px;</string>
    </property>
    <property name="title">
     <string>Seleccione su respuesta:</string>
    </property>
    <widget class="QRadioButton" name="rb_a">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>711</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid black;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_b">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>711</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_c">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>711</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 3px solid black;
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_d">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>711</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid black;
border-radius: 20px;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_empezar">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
border: 5px solid black;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Iniciar Examen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_min">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_seg">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-5</y>
      <width>801</width>
      <height>841</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/Examen.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_respuestas">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>721</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Bright</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 10px solid black;
background-color: rgb(85, 170, 127);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>lbl_pregunta</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_2</zorder>
   <zorder>lbl_respuestas</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
